pkgs/top-level/all-packages.nix

```diff
@@ -13024,7 +13015,7 @@ with pkgs;
   jabref = callPackage ../applications/office/jabref {
     jdk = jdk21.override {
```

Comment:
Combining JDK 21 and OpenJFX 24 seems a bit sketchy. But it previously combined JDK 21 and OpenJFX 23, so this upgrade can't make it worse.